tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+
+@pytest.fixture(scope='session')
+def capabilities(capabilities):
+    capabilities.setdefault('tags', []).append('socorro')
+    return capabilities
+
+
+@pytest.fixture
+def existing_user(variables):
+    return variables['users']['default']
+
+
+@pytest.fixture
+def selenium(selenium):
+    selenium.implicitly_wait(10)
+    selenium.maximize_window()
```

Comment:
Is there a value you'd suggest using? I was using <a href="https://github.com/mozilla/Affiliates-Tests/commit/c6031c9f90f5debd6bc64aec41c8f6a7bdacfaae#diff-484462fced51d1a06b1d93b4a44dd535R30">Affilates Tests</a> as a partial template.